libbeat/_meta/config.reference.yml.tmpl

```diff
@@ -1125,6 +1125,11 @@ logging.files:
 # output configuration, so if you have the Elasticsearch output configured such
 # that it is pointing to your Elasticsearch monitoring cluster, you can simply
 # uncomment the following line.
+#
+# Deprecation warning: The use of monitoring.elasticsearch settings are discouraged and will be
+# removed in 8.0. Please use the `beat-xpack` module in Metricbeat to send Beats monitoring data
```

Comment:
s/`8.0`/`8.0.0`/. We're trying to use complete version numbers everywhere across the codebase (see https://github.com/elastic/beats/pull/14636).

Reply:
Also, consider replacing `Beats` with `{{.BeatName | title}` so the comment is "personalized" for each Beat.

Reply:
Ahh, good to know. Will fix. Thanks!